code/modules/projectiles/ammo_datums.dm

```diff
@@ -3384,12 +3384,6 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	flame_radius(2, T, burn_duration = 9, colour = "blue")
 	playsound(T, 'sound/weapons/guns/fire/flamethrower2.ogg', 35, 1, 4)
 
-/datum/ammo/energy/plasma/cannon_standard
-	damage = 20
-	penetration = 15
-	sundering = 0.75
-	damage_falloff = 0.75
-
```

Comment:
? Why are you deleting this ammo type when it is still used by the cannon's mode list.

Also more balance during freeze woo